extensions/sql-database-projects/resources/templates/newSdkSqlProjectTemplate.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+  <Sdk Name="Microsoft.Build.Sql" Version="0.1.1-alpha" />
+  <PropertyGroup>
+    <Name>@@PROJECT_NAME@@</Name>
+    <ProjectGuid>{@@PROJECT_GUID@@}</ProjectGuid>
+    <DSP>Microsoft.Data.Tools.Schema.Sql.Sql@@PROJECT_DSP@@DatabaseSchemaProvider</DSP>
+    <ModelCollation>1033, CI</ModelCollation>
+  </PropertyGroup>
+  <Target Name="BeforeBuild">
+    <Delete Files="$(BaseIntermediateOutputPath)\project.assets.json" />
```

Comment:
This BeforeBuild target was added in https://github.com/microsoft/azuredatastudio/pull/13736 so that projects can be built in SSDT without having to do a clean first, but since SDK style projects aren't compatible with SSDT yet, I'm not sure if this should be kept in the SDK project template.